site/source/docs/tools_reference/emcc.rst

```diff
@@ -282,6 +282,9 @@ Options that are modified or new in *emcc* are listed below:
 ``--exclude-file <name>``
 	Files and directories to be excluded from :ref:`--embed-file <emcc-embed-file>` and :ref:`--preload-file <emcc-preload-file>`. Wildcards (*) are supported.
 
+``--fully-prepare-files``
+	Tells the file packager to not only load the preloaded files, but also prepare them, similarly to calling ``emscripten_async_prepare()``: This also does preparation tasks like decoding images using the browser codecs.
```

Comment:
I think it would be best to remove `emscripten_async_prepare` to avoid confusion, if we go with that name - which I like the more I think about it.

It's a compile-time error, so while it does require a change, it seems worth it. We could use a C macro so that the compile-time error is clear.